README.md

```diff
@@ -1,5 +1,5 @@
 # powerbi-visuals-gantt
-[![Build Status](https://travis-ci.org/Microsoft/powerbi-visuals-gantt.svg?branch=master)](https://travis-ci.org/Microsoft/powerbi-visuals-gantt) [![Coverage Status](https://coveralls.io/repos/github/Microsoft/powerbi-visuals-gantt/badge.svg?branch=master)](https://coveralls.io/github/Microsoft/powerbi-visuals-gantt?branch=master)
+[![Build Status](https://travis-ci.org/Microsoft/powerbi-visuals-gantt.svg?branch=master)](https://travis-ci.org/Microsoft/powerbi-visuals-gantt)
```

Comment:
![image](https://github.com/microsoft/powerbi-visuals-gantt/assets/19713326/a565b034-fa08-4add-bf57-f86e96802393)

build status is not shown properly, remove it from Readme